docker-compose.yml

```diff
@@ -53,13 +53,7 @@ services:
       - weaviate
     restart: on-failure:0
   janus:
-    image: janusgraph:0.2.0-hadoop2
-    build:
-      dockerfile: Dockerfile.janus
-      args:
-        version: 0.2.0
-        hadoop: hadoop2
-      context: ./tools/dev/janusgraph-docker/
+    image: creativesoftwarefdn/janusgraph-docker
```

Comment:
Could you update this to refer to a tagged version?

Reply:
I guess the easiest would be to tag it with the version of janusgraph?

Reply:
see: https://github.com/creativesoftwarefdn/janusgraph-docker/issues/5

Reply:
fixed